### ISSUE TITLE
setuptools 65 breaks crick

### ISSUE THREAD

**Issue by crusaderky** [dask/crick#24]
crick installation has started failing since setuptools 65.0.0 was released:

```yaml
name: test
channels:
  - conda-forge
  - defaults
dependencies:
  - python=3.8
  - pip
  - cython
  - numpy
  - pip:
      - git+https://github.com/jcrist/crick
```
conda crashes on Linux, citing a Windows-only module:
```
      building 'crick.tdigest' extension
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-req-build-5gm9pgaz/setup.py", line 58, in <module>
          setup(name='crick',
        [...]
        File "/home/crusaderky/miniconda3/envs/test/lib/python3.8/site-packages/setuptools/_distutils/command/build_ext.py", line 547, in build_extension
          objects = self.compiler.compile(
        File "/home/crusaderky/miniconda3/envs/test/lib/python3.8/site-packages/numpy/distutils/ccompiler.py", line 89, in <lambda>
          m = lambda self, *args, **kw: func(self, *args, **kw)
        File "/home/crusaderky/miniconda3/envs/test/lib/python3.8/site-packages/numpy/distutils/ccompiler.py", line 272, in CCompiler_compile
          jobs = get_num_build_jobs()
        File "/home/crusaderky/miniconda3/envs/test/lib/python3.8/site-packages/numpy/distutils/misc_util.py", line 91, in get_num_build_jobs
          from numpy.distutils.core import get_distribution
        File "/home/crusaderky/miniconda3/envs/test/lib/python3.8/site-packages/numpy/distutils/core.py", line 24, in <module>
          from numpy.distutils.command import config, config_compiler, \
        File "/home/crusaderky/miniconda3/envs/test/lib/python3.8/site-packages/numpy/distutils/command/config.py", line 19, in <module>
          from numpy.distutils.mingw32ccompiler import generate_manifest
        File "/home/crusaderky/miniconda3/envs/test/lib/python3.8/site-packages/numpy/distutils/mingw32ccompiler.py", line 28, in <module>
          from distutils.msvccompiler import get_build_version as get_build_msvc_version
      ModuleNotFoundError: No module named 'distutils.msvccompiler'
      [end of output]
```

### Workaround
Add `- setuptools <65` to the conda environment

**Comment by graingert** [comment]
@crusaderky this should be fixed in setuptools 65.0.2: https://github.com/pyodide/pyodide/issues/2971#issuecomment-1225104511

**Comment by crusaderky** [comment]
Confirmed I can no longer reproduce
